CONTRIBUTING.md

```diff
@@ -80,6 +80,7 @@ dependency_overrides:
 Run the following steps, from the `src` directory created in the steps above:
 
  * `gclient sync` to update your dependencies.
+ * `git pull upstream master` in `src/flutter` to update the Flutter Engine repo.
```

Comment:
We should flip these steps.  You usually want to update with git before syncing with gclient